tests/observability/metrics/conftest.py

```diff
@@ -916,3 +917,18 @@ def generated_api_deprecated_requests(prometheus):
     for _ in range(COUNT_FIVE):
         len(list(VirtualMachine.get()))
     return initial_metric_value + COUNT_FIVE
+
+
+@pytest.fixture()
+def storage_class_info_for_testing(admin_client):
```

Comment:
If this is the expected labels, then please rename the fixture to indicate that.